introspection-engine/connectors/sql-introspection-connector/src/calculate_datamodel.rs

```diff
@@ -170,13 +170,14 @@ mod tests {
                     fields: vec![Field::ScalarField(ScalarField {
                         name: "primary".to_string(),
                         arity: FieldArity::Required,
-                        field_type: FieldType::NativeType(
+                        field_type: FieldType::Scalar(
                             ScalarType::Int,
-                            NativeTypeInstance {
+                            None,
+                            Some(NativeTypeInstance {
```

Comment:
Many of the (+) lines are from these tests.

Reply:
Maybe we should sent the tests to Florida to have their retirement.